cg/cli/set/cases.py

```diff
@@ -59,26 +57,26 @@ def families(
 ):
     """Set values on many families at the same time"""
     store: Store = context.obj.status_db
-    cases: List[Family] = _get_cases(identifiers, store)
+    case_list: List[Family] = _get_cases(identifiers, store)
 
-    if not cases:
+    if not case_list:
         LOG.error("No cases to alter!")
         raise click.Abort
 
     LOG.info("Would alter cases:")
 
-    for case in cases:
-        LOG.info(case)
+    for case_obj in case_list:
+        LOG.info(case_obj)
 
     if not (click.confirm(CONFIRM)):
         raise click.Abort
 
-    for case in cases:
+    for case_obj in case_list:
         context.invoke(
-            family,
+            case,
             action=action,
             priority=priority,
             panel_abbreviations=panel_abbreviations,
-            family_id=case.internal_id,
+            family_id=case_obj.internal_id,
             customer_id=customer_id,
         )
```

Comment:
See comment above

Reply:
in here is the only case I can't change it because `case` is also the function.